e2e/tests/features/201-message-control.spec.ts

```diff
@@ -20,7 +20,7 @@ test('message control sends messages to screens', async ({ context }) => {
   await editorPage.getByPlaceholder('Shown in lower third').fill('testing lower');
   await editorPage.getByRole('button', { name: /toggle lower third message/i }).click();
 
-  await featurePage.goto('http://localhost:4001/lower');
+  await featurePage.goto('http://localhost:4001/lower?trigger=manual&bottom-src=lowerMsg');
```

Comment:
how does the manual trigger work? I dont understand how the users control it

Reply:
by using the show hide button in the message controller

Reply:
I see, so we remove the other text line and use only the lower?
You have more experience on this than me so I will follow your lead

I wonder whether triggering the in and out of you could be annoying.
How about a button that says trigger which would make it appear and disappear after the timeout period

Reply:
> I wonder whether triggering the in and out of you could be annoying.

It could but I think it being a toggle is the most flexible and don't require us to add more settings and buttons
and if uses want it to auto animate out that is super easy to set up in companion
I could make it one of the presets